source/releasenotes/2024-03-26-terminus-github-action-sessions-cache.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Terminus Github Action Session Caching
+published_date: "2024-03-26"
+categories: [new-feature, tools-apis]
+---
+We've updated our Terminus GitHub Action with the ability to share sessions across jobs to simplify and reduce the authentication overhead including any associated rate limiting when authenticating their machine token. In addition, these sessions are encrypted prior to being cached. using industry standard AES-256 encryption, PBKDF2 (Password-Based Key Derivation Function 2) algorithm, and increased iterations when deriving the encryption key. We've also updated to the latest versions of `actions/checkout` and `actions/cache`.
```

Comment:
Worth linking to the GitHub repo for the actions. Without linking, you might not know what this is or that it even exists prior to the release note.

Reply:
done, thanks!